### PR TITLE
Clarify `WildcardImport` rule configuration

### PR DESCRIPTION
**Pull request by AlexCzar**
Add information about conflict between `formatting:` `NoWildcardImports` and `style:` `WildcardImport` rules.

There is a conflict between `formatting:` `NoWildcardImports` and `style:` `WildcardImport` , effectively the latter and its configuration do not matter when former is enabled. This PR adds a note about the fact to the documentation.

### CODE REVIEW

File: docs/pages/documentation/style.md [hunk 1]
```diff
@@ -1410,6 +1410,11 @@ Library updates can introduce naming clashes with your own classes which might r
    Define a whitelist of package names that should be allowed to be imported
 with wildcard imports.
 
+**NOTE:** This rule conflicts with `NoWildcardImports` formatting rule.
```
Comment (cortinico) [inline docs/pages/documentation/style.md]: I do believe that _conflicts_ here is probably not the best term.

I would rephrase with something like:

```suggestion
**NOTE:** This rule is similar to the `NoWildcardImports` formatting rule (a wrapper of a KtLint rule).
If you need to specify a `excludeImports` configuration, please consider disabling the other rule.
```

Or something like that.

Comment (AlexCzar) [reply docs/pages/documentation/style.md]: I like the wrapper clarification suggestion.
Regarding conflict/similarity: this pull request was born exactly because of the conflict between these two rules and confusion surrounding it. Yes they are indeed similar, but they are also in conflict and the most restrictive one wins.
